generate-community/generate_community.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+# Switch to script's directory, letting it be called from any folder.
+cd $(dirname $0)
+
+# Download a copy of the Bevy community repository.
+# FIXME: Can this be shortened to git clone --depth=1?
```

Comment:
In my quick testing, using the current method only downloads 0.02 KB less of data. I have no idea what that data is, though. Are the small performance benefits worth a more complicated script?

Reply:
Probably not currently no.